README.md

```diff
@@ -29,16 +29,17 @@ There are also some core tools you will need to complete this journey. If you do
  - [Apache Maven 3.5.0](https://maven.apache.org/): A build automation tool primarily used for Java projects.
  - [Java JDK 1.8.0_144](http://www.oracle.com/technetwork/java/javase/downloads/index.html): A software development environment used for developing Java applications.
 
-## Steps
+# Steps
```

Comment:
do we normally use a single '#' here? i though we used double #s here

Reply:
I was looking at some existing journeys and that was what they used (see social CRM). The journey docs didn't seem to be clear on this.

Reply:
looks like you're right, my bad: https://pages.github.ibm.com/developer-journeys/journey-docs/resources/readme_template/